### PR TITLE
"Fix: Align Homepage Searchbar"

### LINKED ISSUE #1624: Search bar alignment issue on homepage.
### Browser

- [X] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [ ] Windows
- [X] Linux
- [ ] macOS
- [ ] Other/not relevant

### Description

The search bar on homepage seems to be misplaced. Aligning it with the icon grid would look better.

### Steps to reproduce

image
The search bar should be shifted to right.

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

### PR DESCRIPTION
**Pull request by SwastikKasera**
closes #1624

## What is the purpose of this pull request?

- [ ] New Icon
- [x] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
This PR fixes search bar alignment on the homepage

Current:
Screenshot 2023-10-22 at 09-52-51 Lucide Icons

Fixed:
Screenshot 2023-10-22 at 20-10-52 Lucide Icons

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: docs/.vitepress/theme/components/home/HomeHeroIconsCard.vue [hunk 1]
```diff
@@ -143,7 +143,7 @@ onBeforeUnmount(() => {
   .search-box {
     top: unset;
     bottom: -24px;
-    left: -24px;
+    /* left: -24px; */
```
Comment (jguddas) [inline docs/.vitepress/theme/components/home/HomeHeroIconsCard.vue:146]:
```suggestion
    left: -24px;
    width: calc(100% + 48px);
```
image

**Comment by jguddas** [comment]
The fixed screenshot is of the old version without the `width` with `width: calc(100% + 48px)` we now have the search overflow on both sides.

**Comment by SwastikKasera** [comment]
Hi, @jguddas, I have implemented your suggestion. Should I have to update something?

**Comment by jguddas** [comment]
I think it's fixed, let's see what @karsa-mistmere thinks.

**Review by jguddas — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
I'm fairly sure that this offset was in fact a deliberate design choice on @ericfennis' part, in line with similarly styled elements, e.g. the as of now unimplemented but planned "See this icon in action" showcase:
image

**Comment by ericfennis** [comment]
@karsa-mistmere Yeah it was by design, but for some people it looks like it is misaligned so I'm open to suggestions. See comment in #1624. 

But I'm not sure if this looks better.

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere Yeah it was by design, but for some people it looks like it is misaligned so I'm open to suggestions. See comment in #1624.
> 
> But I'm not sure if this looks better.

Out of the three versions, I prefer the current, if we really want to change it, personally, I think the version in this comment is the better alternative:
https://github.com/lucide-icons/lucide/issues/1624#issuecomment-1774670651
Screenshot 2023-10-23 at 13-49-22 Lucide Icons

**Comment by SwastikKasera** [comment]
So, can I create the pull request for this. @karsa-mistmere ?

**Comment by karsa-mistmere** [comment]
I'm voting to just leave it as is, but if we change it, I prefer the version above. @ericfennis and @jguddas have the final word.

**Comment by ericfennis** [comment]
I'm with @karsa-mistmere for this one. So leave it as it is for now. When we add new things to the homepage, we can revise this.

**Pull request closed without merging**